src/main/java/com/switchfully/parkshark_2024_10/user/UserRepository.java

```diff
@@ -2,11 +2,16 @@
 
 
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
 import org.springframework.stereotype.Repository;
 
 @Repository
 public interface UserRepository extends JpaRepository<Person, Long> {
 
+    @Query("SELECT p FROM Person p WHERE p.email = :email AND p.password = :password")
+    Person findByEmailAndPassword(@Param("email") String email, @Param("password") String password);
 
-    Person findByEmailAndPassword(String email, String password);
+    @Query("SELECT p FROM Person p WHERE concat(p.firstName, ' ', p.lastName) like concat('%', :name, '%')")
+    Person findByName(@Param("name") String name);
```

Comment:
Should we add tests for these two methods as well? These use custom Queries so it might be good idea to add a test for both. If ever the query changes the tests will make us aware of that.

Reply:
Why did you add the quesries?

Reply:
The queries were added because the tests failed if I didn't.